vizro-core/changelog.d/20230927_081254_petar_pejovic_actions_unit_tests.md

```diff
@@ -0,0 +1,42 @@
+<!--
```

Comment:
Should this file exist if it has nothing in it? (Sounds like a deep philosophical question...)

Reply:
If you try, one of the Github checks will fail, so this file should exist 😄. (Just kidding) 
From a philosophical point of view it sounds fine to me to get contributors to create this even if they don't change anything worth mentioning. I see this compulsion as a reminder to contributors to be aware that they need to describe the changes if they are worth mentioning.
What do you think @maxschulz-COL?

Reply:
Yes! The idea @antonymilne is that its obligatory to create such a file to be aware that it may need writing. So far I think it has been a positive influence, and since its really easy to create (`hatch run docs:changelog`) its not really a burden

Reply:
Yup, and maybe one extra cool bit is that we have an unreleased section at the op: https://github.com/mckinsey/vizro/blob/main/vizro-core/CHANGELOG.md#unreleased

There we refer to the folder containing the fragments, so people can browse what branch/PR introduced what chagnes.

Reply:
Ah ok, nice! Understanding how the changelog works is on my list of things to catch up on but this sounds really great 👍